master/reference/kube-controllers/configuration.md

```diff
@@ -45,11 +45,12 @@ configure API access if needed.
 
 The following environment variables can be used to configure the {{site.prodname}} Kubernetes controllers.
 
-| Environment   | Description | Schema |
-| ------------- | ----------- | ------ |
-| `ENABLED_CONTROLLERS` | Which controllers to run | namespace, node, policy, serviceaccount, workloadendpoint |
-| `LOG_LEVEL`     | Minimum log level to be displayed. | debug, info, warning, error |
-| `KUBECONFIG`    | Path to a kubeconfig file for Kubernetes API access | path |
+| Environment   | Description | Schema | Default |
+| ------------- | ----------- | ------ | -------
+| `ENABLED_CONTROLLERS` | Which controllers to run | namespace, node, policy, serviceaccount, workloadendpoint | policy,namespace,serviceaccount,workloadendpoint,node
+| `LOG_LEVEL`     | Minimum log level to be displayed. | debug, info, warning, error | info
+| `KUBECONFIG`    | Path to a kubeconfig file for Kubernetes API access | path | 
+| `SYNC_NODE_LABELS`    | When enabled, Kubernetes node labels will be copied to Calico node objects. | boolean | true
```

Comment:
s/Calico/{{site.prodname}}

Reply:
This one actually isn't referring to `calico/node`, but rather to Calico node resources.